app/mm-redux/selectors/entities/apps.ts

```diff
@@ -4,8 +4,20 @@
 import {createSelector} from 'reselect';
 
 import {AppBinding} from '@mm-redux/types/apps';
+import {Config} from '@mm-redux/types/config';
 import {GlobalState} from '@mm-redux/types/store';
-import {appsEnabled} from '@utils/apps';
+
+import {getConfig} from './general';
+
+export const appsEnabled = createSelector(
+    (state: GlobalState) => getConfig(state),
+    (state: GlobalState) => state.entities.apps.pluginEnabled as boolean,
```

Comment:
Can this be added to the utility? I ask as the utility was already ported to Gekidou, perhaps you should start thinking in how this will work then

Reply:
There are some selectors that got carried over from https://github.com/mattermost/mattermost-mobile/pull/5563. Should these also be ported to utils?

Reply:
up to you! in the end you'll have to port it anyway, so is fine if you want to keep it like this for now